config/v1/types_proxy.go

```diff
@@ -14,9 +14,11 @@ type Proxy struct {
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 	// Spec holds user-settable values for the proxy configuration
 	// +required
-	Spec ProxySpec `json:"spec"`
+	Spec   ProxySpec   `json:"spec"`
+	Status ProxyStatus `json:"status"`
```

Comment:
nit: This usually comes with a docstring.  The boilerplate seems to be:

```go
// status holds observed values from the cluster. They may not be overridden.
// +optional
```

Reply:
Fixed. Thanks!